.github/workflows/image-build.yaml

```diff
@@ -95,21 +162,10 @@ jobs:
           images: ${{ matrix.image.registry }}/${{ matrix.image.repository }}/${{ matrix.image.image }}
           tags: ${{ matrix.image.tags }}
 
-      - name: Build image
```

Comment:
Wait are we still actually building the bundle image anywhere?

Reply:
Yes, it's in it's own job, line 132 (https://github.com/bpfman/bpfman-operator/pull/24/files#diff-502adaf7a495a56eb3ecef9c7f4c295ce0ac343f9b0d4e2fd321910fdf8f58bdR132). I was on the fence on one job or two. There are a handful of common steps (Checkout bpfman, Install Golang, etc), but about several that are specific to bundle (Generate olm bundle on disk, Push to registry) and even more in the Build Image job that don't need to run.

Reply:
I get that it's in it's own job now and that's alright since we don't have to build it for every different platform, however in the https://github.com/bpfman/bpfman-operator/blob/fa94e4f9c4f536fc2cc92d6195c67f48fe64b442/.github/workflows/image-build.yaml#L132 job I don't see where we're building the bundle image